parts/k8s/kubernetesparams.t

```diff
@@ -240,7 +240,7 @@
       }
     },
     "mobyVersion": {
-      "defaultValue": "3.0.8",
+      "defaultValue": "3.0.9",
```

Comment:
you also need to update the VHD script to install 3.0.9 (installdependencies.sh)

Reply:
Fixed... I think?